tools/wptrunner/wptrunner/stability.py

```diff
@@ -343,6 +343,14 @@ def check_stability(logger, repeat_loop=10, repeat_restart=5, chaos_mode=True, m
             write_summary(logger, step_results, "FAIL")
             return 1
 
-        step_results.append((desc, "PASS"))
-
-    write_summary(logger, step_results, "PASS")
+        step_result = "PASS"
+        if not results:
```

Comment:
This can only happen when there are no results, right @jgraham, and not because it's the wrong test type or something? I forget what exactly the steps are, and it's not obvious looking from the code.

Reply:
I'm not sure we want to fail in this case TBH. I'm insufficeintly convinced that we never schedule this job with either no test paths or bogus test paths.